benchmark/buffers/buffer-read.js

```diff
@@ -22,10 +22,10 @@ function main(conf) {
 
   buff.writeDoubleLE(0, 0, noAssert);
   var testFunction = new Function('buff', [
-    "for (var i = 0; i !== " + len + "; i++) {",
-    "  buff." + fn + "(0, " + JSON.stringify(noAssert) + ");",
-    "}"
-  ].join("\n"));
+    'for (var i = 0; i !== ' + len + '; i++) {',
```

Comment:
@mscdex I wanted to be conservative in the change set to keep the diff manageable. And I wanted to avoid anything that might foster any discussion whatsoever about potential performance issues. If you'd prefer I go through and convert to template strings where it seems to make sense, I can do that. If left to my own devices, I'd prefer to either not do it or put that in as a separate independent pull request.